public/index.html

```diff
@@ -3,11 +3,11 @@
   <head>
     <meta charset="utf-8">
     <title>Hoodie Account</title>
+    <link rel="stylesheet" href="/default/css/default.css">
     <script src="/hoodie/client.js"></script>
   </head>
   <body>
-    <p>
-       Open browser console and play with <code>hoodie.account</code>.
-    </p>
+    <div class="generic-loader-wrap"></div>
+    <script src="/default/js/default.js"></script>
```

Comment:
I would put all the HTML into index.html already, and just toggle it depending on the user’s state. Check out this [account.html](https://github.com/hoodiehq/hoodie-app-tracker/blob/master/public/account.html) file from hoodie-app-tracker. It has a `data-account-state=""` property on the `<body>` tag, we set / change the state in [common.js](https://github.com/hoodiehq/hoodie-app-tracker/blob/master/public/assets/js/common.js) and we use the html attribute in CSS to toggle content in [app.css](https://github.com/hoodiehq/hoodie-app-tracker/blob/master/public/assets/app.css#L103-L118)

That helps a lot with a cleaner separation of content (HTML) and behavior (JS)

Hope that makes sense?